zerver/tests/test_email_notifications.py

```diff
@@ -873,10 +873,12 @@ def test_relative_to_full_url(self) -> None:
         self.assertEqual(actual_output, expected_output)
 
         # An uploaded file
-        test_data = '<a href="/user_uploads/2/1f/some_random_value">/user_uploads/2/1f/some_random_value</a>'
+        test_data = '<a href="/user_uploads/{id}/1f/some_random_value">/user_uploads/{id}/1f/some_random_value</a>'
```

Comment:
Let's use `realm_id`, not just `id`, for the variable here -- it's a lot more readable.

Reply:
Fixed here and in test_bugdown.py now.